### PR TITLE
2239: Search result list view

### PR DESCRIPTION
**Pull request by achton**
http://platform.dandigbib.org/issues/2239

### CODE REVIEW

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 3f0c7343ee8bce9cdba5845265eae0235d98a0ef
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 3f0c7343ee8bce9cdba5845265eae0235d98a0ef
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

File: modules/ting_search/ting_search.module [hunk 1]
```diff
@@ -146,6 +151,62 @@ function ting_search_theme($existing, $type, $theme, $path) {
...
+  $element = array();
+
+  foreach ($items as $delta => $item) {
+    switch ($display['type']) {
+
+      case 'ting_author_plain':
+
+        $markup_string = '';
+        if (count($entity->creators)) {
+          $markup_string = t('!author_link', array(
```
Comment (xendk) [inline modules/ting_search/ting_search.module]: t() is unneeded here. Just implode and run the result through check_plain().

File: modules/ting_search/ting_search.module [hunk 2]
```diff
@@ -146,6 +151,62 @@ function ting_search_theme($existing, $type, $theme, $path) {
 }
 
 /**
+ * Implements hook_field_formatter_info().
+ */
+function ting_search_field_formatter_info() {
+  return array(
+    'ting_author_plain' => array(
```
Comment (xendk) [inline modules/ting_search/ting_search.module]: This would make more sense in ting.field.inc.

File: modules/ting_search/ting_search.module [hunk 3]
```diff
@@ -146,6 +151,62 @@ function ting_search_theme($existing, $type, $theme, $path) {
...
+      'label' => t('Plain'),
+      'field types' => array(
+        'ting_author',
+      ),
+    ),
+    'ting_version_year' => array(
```
Comment (xendk) [inline modules/ting_search/ting_search.module]: Also candidate for ting.field.inc.

**Comment by xendk** [comment]
This has been rejected by DDB. Is it actually in use anywhere?

**Comment by xendk** [comment]
@achton ?

**Comment by DDBdrift** [comment]
https://platform.dandigbib.org/issues/2239 | Rejected

**Pull request closed without merging**